Makefile.dev.mk

```diff
@@ -128,3 +129,6 @@ dev/install/minikube: ## Bootstrap: Install Minikube
 
 dev/install/golangci-lint: ## Bootstrap: Install golangci-lint
 	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(GOLANGCI_LINT_DIR) $(GOLANGCI_LINT_VERSION)
+
+dev/install/goimports: ## Bootstrap: Install goimports
```

Comment:
Let's add this target to `dev/tools/all`

Reply:
It has already been added

Reply:
Sorry, missed that